src/main/java/org/broadinstitute/dsm/db/DdpKit.java

```diff
@@ -47,30 +46,20 @@ public DdpKit(String DsmKitRequestId, String kitLabel, String trackingToId, Stri
         this.CEOrdered = CEOrdered;
     }
 
-    public void changeCEOrdered(boolean orderStatus) {
+    public void changeCEOrdered(Connection conn, boolean orderStatus) {
```

Comment:
Passing through the connection so that all operations can go through on a single transaction.